app/views/patients/show.html.erb

```diff
@@ -88,7 +88,7 @@
           </td>
           <td><%= lab.id %></td>
           <td><%= lab.lab_type %></td>
-          <td><%= lab.specimen_collection %></td>
+          <td><%= lab.specimen_collection.to_date.strftime('%m/%d/%Y') %></td>
```

Comment:
At some point, this will be moved into a React component. Until then, we want to show the dates formatted as `MM/DD/YYYY`.